style.css

```diff
@@ -0,0 +1,1547 @@
+    font-size: 30px;
+  }
+
+  /* contact me section start */
+  .contact-main h3 {
+    font-size: 38px;
+    line-height: 50px;
+  }
+
+  /* contact me section end */
+}
+
+@media all and (max-width: 1250px) {
+  /* recent works section :start */
+  .all-projects {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .all-projects .project:nth-child(1) {
+    grid-column: span 2;
+  }
+
+  .project-detail {
+    gap: 20px;
+  }
+
+  .project-detail .mult-intro {
+    font-size: 36px;
+    line-height: 39px;
+  }
+
+  .project-detail p {
+    font-size: 19px;
+    line-height: 25px;
+  }
+
+  /* recent works section :end */
+  .recent-work .recent-title .recent-header {
+    font-size: 25px;
+  }
+
+  /* about me section :start */
+  .about-me {
+    background-size: 30% 20%;
+  }
+
+  /* about me section :end */
+
+  /* contact me section start */
+  .contact-main h3 {
+    font-size: 38px;
+    line-height: 50px;
+  }
+
+  /* contact me section end */
+}
+
+@media all and (max-width: 1024px) {
+  /* main upper section :end */
+  .main-nav {
+    padding-left: 80px;
+    padding-right: 80px;
+  }
+
+  .main-logo a {
+    font-size: 18px;
+  }
+
+  .main-body {
+    background-size: 30% 90%;
+    padding-left: 30px;
+    padding-top: 80px;
+    gap: 25px;
+  }
+
+  .main-container .intro {
+    font-size: 45px;
+    line-height: 72px;
+  }
+
+  .main-container .title {
+    font-size: 26px;
+    line-height: 37px;
+  }
+
+  .main-container .description {
+    font-size: 16px;
+    line-height: 28px;
+  }
+
+  /* main upper section :end */
+
+  /* recent works section :start */
+  .recent-work {
+    padding-right: 70px;
+    padding-left: 70px;
+  }
+
+  /* multi -project start */
+  .all-projects {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .all-projects .project:nth-child(1) {
+    grid-column: span 2;
+  }
+
+  .multi-image {
+    width: 50%;
+  }
+
+  .project-detail {
+    width: 50%;
+    gap: 24px;
+  }
+
+  .project-detail .mult-intro {
+    font-size: 38px;
+    line-height: 39px;
+  }
+
+  .project-detail p {
+    font-size: 19px;
+    line-height: 25px;
+  }
+
+  .project-detail ul {
+    gap: 2px;
+  }
+
+  /* multi -project end */
+
+  /* recent works section :end */
+  .recent-work .recent-title .recent-header {
+    font-size: 20px;
+  }
+
+  /* about me section :start */
+  .about-me {
+    padding-left: 115px;
+    padding-right: 115px;
+  }
+
+  .about-me p {
+    width: 420px;
+  }
+
+  /* about me section :end */
+
+  /* technology section :start */
+  .single-tech .tech-title {
+    gap: 8px;
+  }
+
+  .tech-title h5 {
+    font-size: 28px;
+    line-height: 40px;
+  }
+
+  /* technology section :end */
+
+  /* contact me section start */
+  .contact-main h3 {
+    font-size: 26px;
+    line-height: 38px;
+  }
+
+  .contact-main {
+    padding-left: 60px;
+    padding-right: 72px;
+  }
+
+  /* contact me section end */
+}
+
+@media all and (max-width: 768px) {
+  .main-nav {
```

Comment:
- [ ] nicely done on creating this media query however it has affected the mobile version, I think it is because you are using the max-width for your media screen instead of the `min-width` property, I believe that if you change it, you may be able to recover your mobile version design, take a look at what happens currently

![5](https://user-images.githubusercontent.com/77843050/215044377-ac5581f6-586f-4eb6-8e35-d88bc22f9bfb.gif)

- [ ] Am comparing your design with the original design on the left, please take care of fixing everything as you go